crates/l2/sequencer/mod.rs

```diff
@@ -15,6 +16,7 @@ use l1_watcher::L1Watcher;
 use metrics::MetricsGatherer;
 use proof_coordinator::ProofCoordinator;
 use tokio::task::JoinSet;
+use tokio_util::sync::CancellationToken;
```

Comment:
I forgot we are reexporting CancellationToken on spawned. Let's use that reexport, instead.